documentation/modules/ROOT/pages/connectors/mongodb.adoc

```diff
@@ -282,19 +282,19 @@ Every change event that captures a change to the `customers` collection has the
 [source,json,indent=0]
 ----
   {
-    "schema": { <1>
+    "schema": { // <1>
       "type": "struct",
-      "name": "fulfillment.inventory.customers.Key" <2>
-      "optional": false, <3>
-      "fields": [ <4>
+      "name": "fulfillment.inventory.customers.Key," // <2>
```

Comment:
```suggestion
      "name": "fulfillment.inventory.customers.Key", // <2>
```

Reply:
@Naros @gunnarmorling - Did one of you already fix this typo?

Reply:
Ahh. Nevermind. I just saw your commit Gunnar.  Thanks!